common/utils/appinsights.js

```diff
@@ -0,0 +1,27 @@
+const appInsights = require('applicationinsights')
+const { applicationInsights } = require('../config')
+
+const initApplicationInsights = () => {
+  if (applicationInsights.disabled) {
+    console.log('Application Insights disabled; disable flag set')
```

Comment:
what can we do about logging here? we can't setup the loggers before we have configured app insights.

Reply:
Use the logger.
```
const logger = require('../logging/logger')
logger.info('blah')
```